ansible/roles/openfisca_api_fr/templates/nginx/openfisca-web-api.conf.j2

```diff
@@ -0,0 +1,20 @@
+{# This file will be updated by certbot if SSL is enabled, so that this virtual host listens on port 443. #}
```

Comment:
Good documentation!

Could we still add a [safe cypher setup](https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1d&hsts=false&guideline=5.6), or will certbot do that too? 🙂

Reply:
The certbot package provides `/etc/letsencrypt/options-ssl-nginx.conf`, and the certbot nginx plugin includes it from the virtual hosts it processes.

Here is the contents of the file. I am not sure but I think the content is similar to the link you provided.

```
# This file contains important security parameters. If you modify this file
# manually, Certbot will be unable to automatically provide future security
# updates. Instead, Certbot will print and log an error message with a path to
# the up-to-date file that you will need to refer to when manually updating
# this file.

ssl_session_cache shared:le_nginx_SSL:10m;
ssl_session_timeout 1440m;
ssl_session_tickets off;

ssl_protocols TLSv1.2 TLSv1.3;
ssl_prefer_server_ciphers off;

ssl_ciphers "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-SHA";
```

Reply:
Thanks! Could you please check that the `ssl_ciphers` are the same? 🙂